process/project_proposals.adoc

```diff
@@ -70,8 +70,13 @@ image::incubation-process.png[Incubation process]
 
 === Graduation process
 
-Graduation process follows the pattern of incubation, but the majority of DD should have happened at the incubation phase so it should be a less onerous process for graduation.
-
+. *Submit Graduation Proposal Template*
+   * Project fills out and submits the [graduation proposal template](graduation-proposal-template.md) in a pull request in the [cncf/toc GitHub repo](https://github.com/cncf/toc)
```

Comment:
The links are formatted in markdown but this is an asciidoc file so they don't render right. Here's a quick syntax - https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/

> [graduation proposal template](graduation-proposal-template.md)

should be `link:graduation-proposal-template.md[graduation proposal template]`

> [cncf/toc GitHub repo](https://github.com/cncf/toc)

should be `https://github.com/cncf/toc[cncf/toc GitHub repo]`

Reply:
thank you. I will update.